src/npm/__tests__/index.test.js

```diff
@@ -128,9 +128,6 @@ describe('getDownloads()', () => {
       pkg.downloadsLast30Days.toString()
     );
 
-    // eslint-disable-next-line no-console
-    console.log('downloads', { jest, angular, holmes });
```

Comment:
this log was here to be able to update the values easily if the test fails

Reply:
Jest already provides a diff if you have anything that does match expectation but fair enough :)